core/src/main/java/com/amazon/opendistroforelasticsearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -72,17 +100,202 @@ private FunctionResolver date() {
         impl(nullMissingHandling(DateTimeFunction::exprDate), DATE, TIMESTAMP));
   }
 
+  /**
+   * Specify a start date and add a temporal amount to the date.
+   * The return type depends on the date type and the interval unit. Detailed supported signatures:
+   * (DATE, DATETIME/TIMESTAMP, INTERVAL) -> DATETIME
```

Comment:
do you mean (DATE/DATETIME/TIMESTAMP, INTERVAL) -> DATETIME

Reply:
yes, fixed it.